co-simulation/bundle/src/assembly/resources/scenarios/Town04/carma/carma_config.json

```diff
@@ -0,0 +1,25 @@
+{
+     "updateInterval": 100,
+    "carmaVehicles":[
+{   
+    "routeID": "0",
+    "lane": 1,
+    "position": 0,
+    "departSpeed": 0,
+    "vehicleType": "vehicle.chevrolet.impala",
+    "applications":["org.eclipse.mosaic.app.tutorial.VehicleCommunicationApp"],
```

Comment:
Is this usually part of the scenario? I think this is running the same thing that ODU had running, e.g. a basic MOSAIC Vehicle not a CARMA Platform vehicle right?